PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1351,46 +1351,29 @@ void Notepad_plus::removeEmptyLine(bool isBlankContained)
 	_findReplaceDlg.processAll(ProcessReplaceAll, &env, true);
 }
 
-void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir)
+void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir, const Progress* progress)

```

Comment:
OK, about `PCTSTR` I get it, It makes sense. 
For `const Progress* const progress` it doesn't look that beneficial to me.